timescaledb/how-to-guides/multinode-timescaledb/multinode-ha.md

```diff
@@ -1,34 +1,100 @@
 # High availability (HA) with multi-node
-A multi-node installation can be configured so that each chunk is written to
-multiple data nodes. This natively replicates data at the chunk level, and
-protects against data node failures. It also provides high availability using an
-alternative to fully replicating each data node using.
-
-The data nodes themselves require no additional setup to use native replication,
-but the access node relies on streaming replication by default, so it requires
-some configuration to make it highly available.
-
-When data is written to a chunk, it uses the `replication_factor` setting to
-determine how many nodes it replicates the data to. By default, this is set to
-`1`, so there is no native replication. You can increase this number when you
-create the hypertable. For example, to replicate the data across a total of
-three data nodes:
+
+A multi-node installation of TimescaleDB can be made highly available
+in two different ways:
+
+1. by setting up one or more standbys for each node in the cluster, or
+2. by natively replicating data at the chunk level.
+
+The first option relies on streaming replication and the setup is
+similar to [configuring single-node HA][single-ha], although the
+configuration needs to be applied to each node independently.
+
+The second option relies on built-in capabilities of multi-node
+TimescaleDB to replicate data at the chunk level instead of
+replicating entire data nodes. The access node still relies on a
+streaming replication standby, but the data nodes need no additional
+configuration. Instead, the existing pool of data nodes share
+responsibility to host chunk replicas in order to be able to sustain
+node failures.
+
+There are pros and cons of each approach. Setting up standbys for each
+node in the cluster ensures that standbys are identical at the
+instance level and is a tried and tested method to provide high
+availability. However, it is also a heavy-weight solution and
+essentially requires setting up a mirror cluster.
```

Comment:
```suggestion
There are advantages and disadvantages to each approach. 
Setting up standbys for each node in the cluster ensures that 
standbys are identical at the instance level, and this is a tried
and tested method to provide high availability. However, it also
requires more setting up and maintenance for the mirror cluster.
```

Reply:
"it also requires more setting up" --- sounds a bit weird to me, but I am not a native English speaker. Just want to double-check @Loquacity